### PR TITLE
Don't upload test meta data to description anymore

### PR DESCRIPTION
**Pull request by epaulet**
We no longer need to save test meta-data in the notes section, just regular notes.

### CODE REVIEW

File: lib/rainforest_cli/test_parser.rb [hunk 1]
```diff
@@ -26,8 +26,7 @@ def initialize(file_name)
       @test.browsers = []
     end
 
-    TEST_DATA_FIELDS = [:start_uri, :title, :site_id, :browsers].freeze
-    STEP_DATA_FIELDS = [:redirect].freeze
+    TEST_DATA_FIELDS = [:start_uri, :title, :site_id, :tags, :browsers].freeze
```
Comment (curtis-rainforestqa) [inline lib/rainforest_cli/test_parser.rb]: is the addition of `:tags` intentional here?

File: lib/rainforest_cli/test_parser.rb [hunk 2]
```diff
@@ -37,32 +36,31 @@ def process
...
 
-            # if it's supposed to be a CSV field, split and trim it
-            if CSV_FIELDS.include?(field)
-              @test[field] = @test[field].split(',').map(&:strip).map(&:downcase)
+              # if it's supposed to be a CSV field, split and trim it
+              if CSV_FIELDS.include?(matched_field)
+                @test[matched_field] = @test[matched_field].split(',').map(&:strip).map(&:downcase)
+              end
             end
           end
```
Comment (curtis-rainforestqa) [inline lib/rainforest_cli/test_parser.rb]: any tests we should update for this?

File: lib/rainforest_cli/test_parser.rb (hunk 1 above)
Comment (epaulet) [inline lib/rainforest_cli/test_parser.rb]: Yeah. The only time we use `TEST_DATA_FIELDS` alone is when we concat it with `CSV_FIELDS` anyway.

File: lib/rainforest_cli/test_parser.rb (hunk 2 above)
Comment (epaulet) [inline lib/rainforest_cli/test_parser.rb]: Added some specs for this. I thought they already existed.

File: lib/rainforest_cli/test_parser.rb (hunk 1 above)
Comment (curtis-rainforestqa) [inline lib/rainforest_cli/test_parser.rb]: right on

File: lib/rainforest_cli/test_parser.rb (hunk 2 above)
Comment (curtis-rainforestqa) [inline lib/rainforest_cli/test_parser.rb]: NICE

**Pull request merged**